test/e2e/copy_from_image_test.go

```diff
@@ -195,89 +236,47 @@ func TestCopyAnImageFromARepoToATarThatDoesNotContainNonDistributableLayersButTh
 }
 
 func TestCopyToTarWithoutNonDistributableLayersFlagButContainsANonDistributableLayerShouldPrintAWarningMessage(t *testing.T) {
-	t.Run("environment with internet", func(t *testing.T) {
-		env := BuildEnv(t)
-		imgpkg := Imgpkg{t, Logger{}, env.ImgpkgPath}
-		defer env.Cleanup()
-
-		// general setup
-		testDir := env.Assets.CreateTempFolder("image-to-tar")
-		tarFilePath := filepath.Join(testDir, "image.tar")
+	env := BuildEnv(t)
```

Comment:
Instead of this maybe we can shorten the name of the func and have this `t.Run` with more description

Reply:
This is a good call out. I collapsed 3 "long named tests" into a test called:
"TestCopyRepoToTarAndThenCopyFromTarToRepo"
and then those 3 tests from before are nested t.Runs.